rapids/README.md

```diff
@@ -230,13 +331,14 @@ Dask-cuDF cluster:
 ### Create Dataroc cluster with Dask RAPIDS
 
 Using the `gcloud` command to create a new cluster with this initialization
-action.
+action. Due a anaconda version conflict, we only support dask in dataproc with
+version > 2.0
```

Comment:
I think we can delete this

Reply:
Correct, I made a modification to recommend 2.0 and up. The deployment time is just too long for older images.

Reply:
I've been learning this the hard way... this recommendation seems reasonable.